src/main/java/hudson/plugins/git/GitObject.java

```diff
@@ -15,8 +16,8 @@
 
     private static final long serialVersionUID = 1L;
 
-    ObjectId sha1;
-    String name;
+    final ObjectId sha1;
+    final String name;
```

Comment:
@jglick, @stephenc or @svanoort, does the change from package protected non-final to package protected final cause any risks in migration from version to version?  The object is serializable, and this change did not alter the number, type, or names of the fields, only the access

Reply:
Should not affect downstream plugins (unless they are squatting in the same package name... in which case they deserve what they get ;-) )